actionpack/test/dispatch/show_exceptions_test.rb

```diff
@@ -98,7 +98,7 @@ def call(env)
 
   test "calls custom exceptions app" do
     exceptions_app = lambda do |env|
-      assert_kind_of AbstractController::ActionNotFound, env["action_dispatch.exception"]
+      assert_kind_of ActionView::Template::Error, env["action_dispatch.exception"]
```

Comment:
Not in this case - `ActionView::Template::Error` was a special, and used to store the `original_exception` (basically its `cause`), and `ExceptionWrapper` would use this, rather than the outermost exception. We should be fine for it to change now, though. Just put a comment on the PR explaining.